### PR TITLE
License added

### PR DESCRIPTION
**Pull request by rupertford**
Added a bsd 3-clause license in its own directory and then added it to all Python files.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Science and Technology Failities Council
```
Comment (arporter) [inline LICENSE]: Unfortunately you too have got a typo in your license: "Failities" and, just like mine, it's in every file :-(

**Review by arporter — CHANGES_REQUESTED** [review]
License has been added to all files *apart* from .travis.yml and the 'template' files in the examples. I think the latter is OK but please add it to the travis config file.

Please also correct the typo ("Failities").

**Comment by rupertford** [comment]
Oops, typo has been fixed and license added to the .travis.yml file.

**Review by arporter — APPROVED** [review]

**Comment by arporter** [comment]
Can't see any way of formally doing a second review on here but never mind. The changes are good. Confusingly landscape is saying new errors were found but if I click on 'details' and go to the landscape website it says that no errors were introduced. Anyway, this pull request may now be merged.

**Comment by rupertford** [comment]
I agree the error is strange. I changed the default values in landscape so it would fail if any new errors were found, but as you say, not new errors are reported. Anyway I've changed the settings to fail if the quality goes down (the default is to pass in any case).

**Pull request merged**